### PR TITLE
prov/efa: Add warning when efawin.dll fails to load

### PR DESCRIPTION
**Pull request by wckzhang**

### CODE REVIEW

**Review by wzamazon — APPROVED** [review]

**Review by shijin-aws — APPROVED** [review]

**Comment by wckzhang** [comment]
Acutally FI_LOG_DOMAIN is probably wrong, do you guys want me to change this to FI_LOG_CORE or something?

**Comment by wzamazon** [comment]
bot:aws:retest

File: prov/efa/src/efa_prov.c [hunk 1]
```diff
@@ -65,7 +65,12 @@ int efa_win_lib_initialize(void)
 	* efa_load_efawin_lib function will replace stub ibv_* functions with
 	* functions from efawin dll
 	*/
-	return efa_load_efawin_lib();
+	int err = efa_load_efawin_lib();
+	if (ret) {
```
Comment (wzamazon) [inline prov/efa/src/efa_prov.c:69]: `ret` should have been `err`

Comment (wckzhang) [reply prov/efa/src/efa_prov.c]: Done, I switched from ret to err as a last minute change and didn't check to recompile *facepalm*

**Pull request merged**